Library/Homebrew/os/linux/elf.rb

```diff
@@ -33,7 +33,7 @@ module ELFShim
   private_constant :ARCHITECTURE_POWERPC
   ARCHITECTURE_ARM = 0x28
   private_constant :ARCHITECTURE_ARM
-  ARCHITECTURE_X86_64 = 0x62
+  ARCHITECTURE_X86_64 = 0x3E
```

Comment:
This is needed because checking the `arch` of ELF files on Linux currently returns `:dunno`.